software/rust/en.md

```diff
@@ -0,0 +1,17 @@
++++
+title = "Rust"
+lastmod = "2020-11-22T14:00:30+01:00"
++++
+# Rust
+
+In Solus, the Rust toolchain can be installed in two ways:
```

Comment:
There should be a line above this that acts as a description used in the listing of articles. Then this should start on line 9 (so an empty line between the two).